docs/source/boundaries.rst

```diff
@@ -9,8 +9,24 @@ Simple boundaries for an optimisation can be created using
 More complex types can be made using :class:`LogPDFBoundaries` or a custom
 implementation of the :class:`Boundaries` interface.
 
+Overview:
+
+- :class:`Boundaries`
+- :class:`LogPDFBoundaries`
+- :class:`RectangularBoundaries`
+
+
+Customisable boundaries
```

Comment:
I don't think we should have a sub-header for each class. If the distinction needs to be clearer we could probably modify the CSS instead?

Reply:
Agree with Michael for the sub-header for each class -- looks a bit confusing in fact...

Reply:
Except if we do this for _every_ autoclass, than we could generate the toctree for all without manually typing in in the summary...?